sql/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/view.sql

```diff
@@ -2,8 +2,39 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.cohort_daily_statistics`
 AS
 SELECT
-  *
+  activity_date,
+  cohort_date,
+  normalized_app_name,
+  normalized_channel,
+  normalized_os,
+  normalized_os_version,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_content,
+  attribution_experiment,
+  attribution_variation,
+  num_clients_active_on_day,
+  num_clients_in_cohort
 FROM
   `moz-fx-data-shared-prod.telemetry_derived.cohort_daily_statistics_v1`
 WHERE
   normalized_app_name != "Firefox Desktop"
+UNION ALL
+SELECT
+  submission_date AS activity_date,
+  first_seen_date AS cohort_date,
+  "Firefox Desktop" AS normalized_app_name,
+  normalized_channel,
+  normalized_os,
+  normalized_os_version,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_content,
+  attribution_experiment,
+  NULL AS attribution_variation,
+  num_clients_active_on_day,
+  num_clients_in_cohort
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.desktop_cohort_daily_retention_v1`
```

Comment:
For consistency it's good to always end files with a trailing newline.

Reply:
I had one, it disappeared and I was getting an error, but it cleared up this time
![Screenshot 2023-12-15 at 5 54 22 AM](https://github.com/mozilla/bigquery-etl/assets/92952117/d913ac26-c8c6-4a18-882b-bbf65091a441)